cppcache/integration-test/testSerialization.cpp

```diff
@@ -68,8 +68,7 @@ class OtherType : public DataSerializable {
   CData m_struct;
   int32_t m_classIdToReturn;
 
-  explicit OtherType(int32_t classIdToReturn = g_classIdToReturn)
-      : m_classIdToReturn(classIdToReturn) {
+  explicit OtherType() {
```

Comment:
Why `explicit` here? This is now a default constructer and could never implicitly be invoked.

Reply:
This is preexisting test code. Our change was to remove the classId parameter.

Reply:
Right, and by removing that param you made `explicit` redundant. Please remove.